device-types/Fortinet/FG-2600F.yaml

```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2600F
+slug: fortinet-fg-2600f
+part_number: FG-2600F
+u_height: 2
+is_full_depth: true
+weight: 13.9
+weight_unit: kg
+airflow: front-to-rear
+console-ports:
```

Comment:
Not sure what happened to the power, can you want to add those back as module bays?

Reply:
done